ceph-releases/ALL/rhel8/daemon-base/__ISCSI_PACKAGES__

```diff
@@ -0,0 +1 @@
+tcmu-runner ceph-iscsi-config ceph-iscsi-cli
```

Comment:
```ceph-iscsi``` for RHEL8 -- does this mean we need to create a builder for RHEL8 rpms?

Reply:
:+1: we will have `ceph-iscsi` packages here. (upstream builds TBD)